src/AppBundle/Controller/ArticleController.php

```diff
@@ -6,14 +6,73 @@
 use AppBundle\Entity\ArticleCategory;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Suin\RSSWriter\Channel;
+use Suin\RSSWriter\Feed;
+use Suin\RSSWriter\Item;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 
 class ArticleController extends Controller
 {
     const PER_PAGE = 12;
 
     /**
      * @Route(
+     *     "/feed",
+     *     name="articles_feed"
+     * )
+     * @Method("GET")
+     */
+    public function feedAction()
+    {
+        $articlesRepo = $this->getDoctrine()->getRepository(Article::class);
+
+        $articles = $articlesRepo->findAllForFeed();
+
+        if (empty($articles)) {
+            throw $this->createNotFoundException();
+        }
+
+        $mostRecentArticleDateTimestamp = $articles[0]->getPublishedAt()->format('U');
+
+        $feed = new Feed();
```

Comment:
This action is too big, you should create a service dedicated to building the feed.